examples/serialization/load.py

```diff
@@ -0,0 +1,28 @@
+import chainer
+import chainer.functions as F
+import chainer.links as L
+
+n_in = 784
+n_units = 100
+n_out = 10
+
+# Network definition
+class MLP(chainer.Chain):
+
+    def __init__(self, n_in, n_units, n_out):
+        super(MLP, self).__init__()
+        with self.init_scope():
```

Comment:
How about demonstrating `add_persistent`?

Reply:
That's nice idea. I added it.